packages/app/src/cli/services/dev/update-extension.ts

```diff
@@ -68,6 +65,24 @@ export async function updateExtensionDraft({
   }
 }
 
+export async function updateMultipleExtensionDrafts({
+  extension,
+  developerPlatformClient,
+  apiKey,
+  registrationId,
+  stdout,
+  stderr,
+}: UpdateExtensionDraftOptions) {
+  const registrationIdArr = registrationId.split(',')
+  const configArr = await multipleConfigs(extension)
+
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  registrationIdArr.forEach(async (registrationId, index) => {
+    const config = configArr[index] as object
```

Comment:
_also_ a hack - I'm not sure how we want to store the `config` in the draft registrations? This _seems_ to work but still 😬